src/js/course-list.js

```diff
@@ -7,8 +7,9 @@ $(() => {
     /*
         Click event to sort the course list
      */
-    $('#course-list th:not(:last)').on('click', function() {
-        var isAscending = (isDescending = false);
+    $('#course-list th:not(:last)').on('click', function () {
+        var isAscending = false,
+            isDescending = false;
```

Comment:
@vatz88 will need you to approve this. This is the only change I made, the rest was done by the prettier.

Reply:
Approving, I don't really remember the code. Hope you have manually tested it.

Reply:
When was this bug introduced? Or it has always existed?

Reply:
@vatz88 I'm not sure actually, I think it always existed. The reason I didn't catch it is because for some reason I can't reproduce this locally, it works fine.

Reply:
I think it could be due to auto-formatting.

I'd actually just prefer writing it as this, and use let/const over var

```
var isAscending = false;
var isDescending = false;
```

Reply:
> I think it could be due to auto-formatting.

Yeah, I think so too. I initially wrote it as `var isAscending = isDescending = false`. But this two might give other issues so it's just better to separate them.

> I'd actually just prefer writing it as this, and use let/const over var
> 
> ```
> var isAscending = false;
> var isDescending = false;
> ```

We can open a separate PR that refactors everything and removes all bad practices. I can try doing this when I have some time, I actually wanted to migrate this project to use some framework like react or svelte. Is there any framework you recommend?